integration-tests/tests/src/utils/import-app.ts

```diff
@@ -72,6 +74,8 @@ export function getDefaultReplacements(name: string): TemplateReplacements {
       };
     } else if (name === "with-db-flx") {
       return {
+        ...replacements,
```

Comment:
Perhaps we need a bit deeper merge of these objects?

Reply:
At least in the case of the base `config.json`, the object is flat

Reply:
Never mind - i assumed this was provided as an argument, didn't read all the code above.